<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageLabel</class>
 <widget class="QWidget" name="ImageLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout"/>
  <action name="setMinXAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set &amp;Min X</string>
   </property>
  </action>
  <action name="setMinYAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set M&amp;in Y</string>
   </property>
  </action>
  <action name="setMaxXAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Max &amp;X</string>
   </property>
  </action>
  <action name="setMaxYAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Max &amp;Y</string>
   </property>
  </action>
  <action name="showGridAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Grid</string>
   </property>
  </action>
  <action name="xAxisLogarithmicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>X Axis &amp;Logarithmic</string>
   </property>
  </action>
  <action name="yAxisLogarithmicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Y Axis L&amp;ogarithmic</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
